modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsPerformanceByStep.testcase

```diff
@@ -0,0 +1,274 @@
+	@ignore = "Test Stub"
+	@priority = "3"
+	test PageDisplaysSpecificErrorStateOnUnexpectedError {
+
+		// TODO LRQA-67669 PageDisplaysSpecificErrorStateOnUnexpectedError pending implementation
+
+	}
+
+	@description = "LPS-100677 - Verify the page is providing loading feedback when the table is being updated according to the search queries"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test PageProvidesLoadingFeedbackOnTableUpdate {
+
+		// TODO LRQA-67670 PageProvidesLoadingFeedbackOnTableUpdate pending implementation
+
+	}
+
+	@description = "LPS-100677 - Verify that the search is an approximate match (aka (LIKE) search type or (character by character comparison))"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test SearchIsAnApproximateMatch {
+
+		// TODO LRQA-67672 SearchIsAnApproximateMatch pending implementation
+
+	}
+
+	@description = "LPS-97667 - Verify that when there are tasks with average time smaller than 1 minute the average completion time is displayed"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test TasksWithAverageTimeSmallerThan1MinAreDisplayed {
+
+		// TODO LRQA-67668 TasksWithAverageTimeSmallerThan1MinAreDisplayed pending implementation
+
+	}
+
+	@description = "LPS-97667 - Verify that, when displaying time, the time units are shortened by using the biggest measurement possible, and use up to 2 units per time, with (minute) as smallest value and (days) as biggest unit"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test TimeUnitsAreShortenedByBiggestMeasurementPossible {
+
+		// TODO LRQA-67671 TimeUnitsAreShortenedByBiggestMeasurementPossible pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the To date is greater than the From date"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test ToDateMustBeGreaterThanFromDate {
+
+		// TODO LRQA-67723 ToDateMustBeGreaterThanFromDate pending implementation
+
+	}
+
+	@description = "LPS-100677 - Verify that searching steps containing non-ascii chars is possible"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test UserCanSearchNonAsciiChars {
+
+		// TODO LRQA-67675 UserCanSearchNonAsciiChars pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the card (Performance by Step), when it has 6 or more entries, the table should provide a vertical scroll"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test WhenCard6OrMoreEntriesTableProvideVerticalScroll {
+
+		// TODO LRQA-67679 WhenCard6OrMoreEntriesTableProvideVerticalScroll pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the card (Performance by Step), when it has 5 or fewer entries, the table should not scroll"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test WhenCardHasUpTo5EntriesTableDoNotScroll {
+
+		// TODO LRQA-67677 WhenCardHasUpTo5EntriesTableDoNotScroll pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when an SLA is deleted, the data in (Performance by Step) is still part of the average calculation"
```

Comment:
Please, remove the parenthesis from (Performance by Step)
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.